core/src/test/java/alt/testcontainers/images/OutOfPackageImagePullPolicyTest.java

```diff
@@ -0,0 +1,27 @@
+package alt.testcontainers.images;
+
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.startupcheck.OneShotStartupCheckStrategy;
+import org.testcontainers.images.AbstractImagePullPolicy;
+import org.testcontainers.images.ImageData;
+import org.testcontainers.utility.DockerImageName;
+
+public class OutOfPackageImagePullPolicyTest {
+    @Test
+    public void shouldSupportCustomPoliciesOutOfTestContainersPackage() {
+        try (
+            GenericContainer<?> container = new GenericContainer<>("hello-world:latest")
+                .withImagePullPolicy(new AbstractImagePullPolicy() {
+                    @Override
+                    protected boolean shouldPullCached(DockerImageName imageName, ImageData localImageData) {
+                        return true;
```

Comment:
```suggestion
                        return false;
```

Let's not do unnecessary pulls

Reply:
Fixed. Thanks!